Mathlib/CategoryTheory/Sites/Equivalence.lean

```diff
@@ -34,16 +34,95 @@ sufficiently small limits in the sheaf category on the essentially small site.
 
 -/
 
-universe u
+universe w' w v u
 
 namespace CategoryTheory
 
-open Functor Limits GrothendieckTopology
+open Functor Limits GrothendieckTopology Presheaf
 
 variable {C : Type*} [Category C] (J : GrothendieckTopology C)
 variable {D : Type*} [Category D] (K : GrothendieckTopology D) (e : C ≌ D)
 variable (A : Type*) [Category A]
 
+namespace Presheaf
+
+variable (H : C ⥤ D) {F G : Dᵒᵖ ⥤ A} (f : F ⟶ G)
+
+lemma _root_.CategoryTheory.GrothendieckTopology.W_iff_whisker
```

Comment:
For the transport of `WEqualsLocallyBijective`, I intended an approach where we do not rely on `HasWeakSheafify` and existence of limits, but on the very definition of `W`, taking into consideration that on both equivalent sites, we have equivalent categories of sheaves in order to test the `W` condition.